Studentquiz: Update css for displaying error form-group

JBThong opened the pull request:
Hi Tim,

I have created the commit to change the approach CSS in form-group filter.
As I have discussed with you, we have no choices in the situation so I have set CSS important but it makes the cibot warning. I think we ignore the cibot warning in this case.

Could you please help me to review it?
Thanks

![image](https://user-images.githubusercontent.com/32395146/163513103-55b2994b-21ba-446d-bcdd-9c95522da231.png)

dionysius commented:
Oh, I could have a look at that as well, but easter is around the corner and I have no development setup ready.

Rule of thumb: the most specific css path wins, so if we add a better path we might get rid of the !important part. But maybe you've already discussed that.

Although this task is impossible if the previous style has already !important, which might be the case.

JBThong commented:
Yes, thanks @dionysius,

The align-items-center class has !important CSS. I have discussed with Tim about this. We will back to it after the easter.

Happy Easter!! :))

timhunt commented:
If we are going to do !important, then we need to ensure it does not break CI (https://stylelint.io/user-guide/ignore-code/)

However, do we really need to override this? What happens if we just leave the standard Boostrap CSS here?

timhunt commented on styles.css:
```diff
@@ -822,15 +822,14 @@
-    .path-mod-studentquiz #form-advanced-div .form-control-feedback.invalid-feedback {
-        margin-top: 4.5em;
-    }
+/* Specific CSS override align-items-center, we need to prevent align-items: center in the form-group. */
+#fgroup_id_rate_grp div.align-items-center,
+#fgroup_id_rate_grp div.align-items-center,
+#fgroup_id_difficultylevel_grp div.align-items-center,
+#fgroup_id_publiccomment_grp div.align-items-center,
+#fgroup_id_myrate_grp div.align-items-center,
+#fgroup_id_myattempts_grp div.align-items-center,
+#fgroup_id_mydifficulty_grp div.align-items-center {
```
We must make sure this style rule does not break layout in other plugins.

Each selector should start with .path-mod-studentquiz or similar (assuming that we really cannot avoid having this.)

JBThong commented:
Thanks for your feedback, Tim

I have added .path-mod-studentquiz and updated the style lint to ignore the important!

Could you please help me to review it again?
Thanks,

timhunt commented:
Thanks Thong. Merged.
